elide-core/src/main/java/com/yahoo/elide/security/executors/ActivePermissionExecutor.java

```diff
@@ -197,6 +200,29 @@ public ActivePermissionExecutor(final com.yahoo.elide.core.RequestScope requestS
     }
 
     /**
+     * Get permission filter on an entity.
+     *
+     * @param resourceClass Resource class
+     * @param annotationClass Annotation class
+     * @param <A> type parameter
+     */
+    public <A extends Annotation> FilterExpression getPermissionFilter(Class<?> resourceClass,
+                                                                       Class<A> annotationClass) {
```

Comment:
This will only apply for ReadPermission.  I don't think we need to pass annotationClass here.